### PR TITLE
portfolio-mobile-finish-version

### PR DESCRIPTION
**Pull request by elad237**
initial file structure setup
Added a ReadMe file
Added an html file
Added CSS folder
Added image and img folders
install linters

### CODE REVIEW

**Review by TadesseDev — COMMENTED** [review]
### Hi @elad237 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

yPkfDFf
## Invalid Code Review Request 🛑🛑

You have submitted a project, that does not set up linters correctly. Kindly, make sure linter are set up and all linters are passing before the next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+  text-align: center;
+  color: #172b4d;
+}
+
+.about-content-text {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  color: #172b4d;
+}
```
Comment (Gopxfs) [inline css/style.css]: Please change your about-section font family to fit with the one on Figma.

image

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+/* About Style
+------------------------------------ */
+.about-wrapper > * {
+  padding-inline: 20px;
+  text-align: center;
+}
+
+.about-content > * + * {
+  margin-block-start: 24px;
+}
```
Comment (Gopxfs) [inline css/style.css]: Please add the corresponding background images to your about section, following the Figma design. 😄

Figma design:
image

Your design:
image

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+.project-item-btn:active {
+  background-color: #008552;
+}
+
+/* About Style
+------------------------------------ */
+.about-wrapper > * {
+  padding-inline: 20px;
+  text-align: center;
+}
```
Comment (Gopxfs) [inline css/style.css]: Kindly add more spacing between your sections, as show on Figma:

Figma design:
image

Your design:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+          <a href="#" class="social-links"
+            ><img src="images/icons/angellist.svg" alt="angellist"
+          /></a>
+        </section >
+      </section>
+
+      <section class="portfolio-wrapper" id="portfolio">
+        <h2 class="portfolio-header">My Recent Works</h2>
+        <section class="portfolio-project">
+            <figure class="project-item">
```
Comment (Gopxfs) [inline index.html]: As represented in Figma, please add:
- The green line under 'My Recent Works'.
- More spacing between the header and the image.
- The circle containing the down arrow. 😉

Figma design:
image

Your design:
image

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+  height: 600px;
+  padding: 10px;
+  align-items: center;
+}
+
+/* styling the social media section */
+.social-media {
+  display: flex;
+  width: 70%;
+}
```
Comment (Gopxfs) [inline css/style.css]: As shown on Figma, please center your social link icons. 😃

Figma design:
image

Your design:
image

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+.about-tech-items li {
+  padding: 12px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+  background-color: #fff;
+  border-radius: 8px;
+}
```
Comment (Gopxfs) [inline css/style.css:302]: Please add to all the texts of your webpage the correct font family. As shown below, you can check for the text properties on Figma to follow the exact requirements with ease:
image
Currently, you are applying this font family only to your class `headline-box`, but it should be applied to the majority of your texts. 😃 
Please try to stick to the design as much as possible. This is important to make you understand design guidelines and how to work with a client's requirements. Imagine Figma as a client, who is asking to you make a website following the proposed design. Good luck and keep coding! 😄
> Your styling is very well done overall, keep up the good work! 😉

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @elad237 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ Good work with your styling!
✔️ Your code is very well structured. Well done!
✔️ Greate readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+                        <li>html</li>
+                    </ul>
+                    <button class="project-item-btn">See project</button>
+                </figcaption>
+            </figure>
+        </section>
+    </section>
+
+    <!-- ABOUT MYSELF SECTION -->
+    <section class="about-wrapper" id="about">
```
Comment (Olamarx) [inline index.html:147]: - [ ] As the previous reviewer rightly recommended, I would advise you to add the background image of the about me section
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @elad237, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linter passed ✔️ 
- Good use of git and github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (elad237) [reply index.html:147]: hi @Olamarx 
thanks for the code  review.
But I have just one worry I have added the background image as requested by the code reviewer.

https://github.com/elad237/mobile-finish-version/pull/1/files#diff-1fc556f95754ee7e33d91044125c44bb9f750c99be4406756ffb27413adfcaf5R212-R217

The background image displays well on my computer

Comment (Olamarx) [reply index.html:147]: It didn't show on my end. You can request for a second view though.

Comment (elad237) [reply index.html:147]: Hi @Olamarx  
Thanks for the reply.
I have fix the issue.
Should I request for another code review?

Comment (Olamarx) [reply index.html:147]: Sure, if you have fixed it.

Comment (elad237) [reply index.html:147]: Hi @Olamarx  
Just wish to inform you that I have fix the issue

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @elad237        :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making all the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**